### PR TITLE
{compiler}[system/system] NVHPC v22.9 w/ CUDA 11.7.0

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

Requires https://github.com/easybuilders/easybuild-easyblocks/pull/2819

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2819
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bwd-rockylinux-91 - Linux Rocky Linux 9.1 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.9.14
See https://gist.github.com/8aef35eb1cfd5b0251cfc7f3b4986971 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/n/NVHPC/NVHPC-22.9-CUDA-11.7.0.eb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    ('numactl', '2.0.14', '', ('GCCcore', local_gccver)),
+    ('CUDA', '11.7.0', '', SYSTEM),
+]
+
+module_add_cuda = False
+
+# specify default CUDA version that should be used by NVHPC
+# should match one of the CUDA versions that are included with this NVHPC version
+# (see install_components/Linux_x86_64/22.3/cuda/)
+# for NVHPC 22.3, those are: 11.6, 11.0, 10.2;
```
Comment (branfosj) [inline easybuild/easyconfigs/n/NVHPC/NVHPC-22.9-CUDA-11.7.0.eb:41]:
```suggestion
# (see install_components/Linux_x86_64/ver/cuda/) where ver is the NVHPC version
```

The suggestion makes this comment more generic and avoids the need to update it for each version. The alternative is that this is changed to be correct for 22.9.

Similarly for 22.11 in #17228 and 23.1 in #17229

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @SebastianAchilles!

**Pull request merged**